index.html

```diff
@@ -710,6 +710,16 @@ <h2>
 
     </section>
 
+    <section class="informative">
+      <h2>
+        Audience
+      </h2>
+      <p>
+        This specification's primary audience is implementors developing conformant DID methods and implementors of systems and services that wish to interact and interface with DIDs. The intended audience includes, but is not limited to, software architects, data modelers, application developers, services developers, testers, operators, and user experience (UX) specialists. In addition, other people involved in a broad range of standards efforts related to decentralized identity, verifiable credentials, and secure storage may also be interested in reading this specification.
```

Comment:
```suggestion
        This specification's primary audience is implementors developing conformant DID methods and systems and services that wish to interact and interface with DIDs. The intended audience includes, but is not limited to, software architects, data modelers, application developers, services developers, testers, operators, and user experience (UX) specialists. In addition, other people involved in a broad range of standards efforts related to decentralized identity, verifiable credentials, and secure storage may also be interested in reading this specification.
```

Reply:
Change 'may' to 'might' to avoid confusion w/ normative language.

Reply:
A little more tweaking... Note that [implementor has fallen into disuse](https://www.google.com/search?q=implementors+vs+implementers).
```suggestion
        This specification has two primary audiences: implementers of conformant DID methods; and implementers of systems and services that wish to interact and interface with DIDs. The intended audience includes, but is not limited to, software architects, data modelers, application developers, services developers, testers, operators, and user experience (UX) specialists. Other people involved in a broad range of standards efforts related to decentralized identity, verifiable credentials, and secure storage might also be interested in reading this specification.
```

Reply:
LGTM, thanks